src/pages/api/frontend/[...slug].ts

```diff
@@ -1,17 +1,14 @@
 import { NextApiRequest, NextApiResponse } from 'next'
-import absoluteUrl from 'next-absolute-url'
 
-import { fetchContent } from '@/fetcher/serlo-api'
+import { fetchPageData } from '@/fetcher/fetch-page-data'
 
 // Proxy the API Call as GET request to the frontend so that the ZEIT Now CDN is able to cache this
 // We use stale-while-revalidate for that, see also https://zeit.co/docs/v2/network/caching#stale-while-revalidate
 export default async function fetch(req: NextApiRequest, res: NextApiResponse) {
   const slug = req.query.slug as string[]
-  const { origin } = absoluteUrl(req)
-  const data = await fetchContent('/' + slug.join('/'), origin)
-  if (data.error) {
-    console.log(data.error)
```

Comment:
Please log the error here. This way, we can debug weird errors for deployed stuff via Vercel's logs.

Reply:
Ups, wondered who added this `console.log`, so it was you :smile: will readd it :+1: